src/_posts/2020-05-05-what-is-software-modernisation.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+asset-type: post
+name: what-is-software-modernisation
+title: What is Software Modernisation and why do you want to do it?
+title-es: ¿Qué es Software Modernisation y por qué puedes necesitarlo?
+date: 2020-05-05 05:00:00 +00:00
+author: Mashooq Badar
+description: Software Modernisation is about ensuring that software is a strategic asset to enable your business and gain a competitive advantage. It is necessary when a legacy of tactical changes have resulted in software that is holding your business hostage.
```

Comment:
What are we using to guide the length of titles and descriptions?

Reply:
Nothing that I'm aware of, but if this doesn't look good on the page then we can discuss of a way to handle with either truncating text or some other way.